### DISCUSSION TITLE
How to Create Login Page

### DISCUSSION THREAD

**Discussion by ThankVisualJkseo** [David-Moreira/BlazorXTabs#34] [Q&A] [answered]
I am creating a login page.

but the input field is not visible

how to fixed?

My Code is. (App.razor)
----------------------------------
<CascadingAuthenticationState>

    <Router AppAssembly="@typeof(Program).Assembly" PreferExactMatches="@true">
        <Found Context="routeData">
            <XTabsAuthorizeRouteView CloseTabs="true"
                            CloseAllTabs="true"
                            NoTabsNavigatesToHomepage="true"
                            ShowCloseAllTabsButton="false"
                            CloseAllTabsButtonThreshold="2"
                            NewTabSetActive="true"
                            RenderMode="BlazorXTabs.Configuration.RenderMode.Full"
                            RouteData="@routeData"
                            DefaultLayout="@typeof(MainLayout)"
                            OnTabRemoved="TabRemoved"
                            OnTabAdded="TabAdded"
                            JustifiedHeader="false">
            <NotAuthorized>
                <RedirectToLogin/>
            </NotAuthorized>
            </XTabsAuthorizeRouteView>
        </Found>
        <NotFound>
            <LayoutView Layout="@typeof(MainLayout)">
                <p>Sorry, there's nothing at this address.</p>
            </LayoutView>
        </NotFound>
    </Router>
</CascadingAuthenticationState>
----------------------------------

OK Screen
![image](https://user-images.githubusercontent.com/91505660/136924390-33d4fb24-12f0-4839-9c8a-bb33e73676ff.png)

NG Screen
![image](https://user-images.githubusercontent.com/91505660/136924434-23783689-5c70-4b5b-b436-f0bf2892cbd9.png)

Thank you,

**Comment by David-Moreira** [comment]
Hello @ThankVisualJkseo
Your example seems fine and this should work the same as AuthorizeRouteView. I'll let you know as soon as I take a look!

**Reply by David-Moreira** [reply]
Can confirm it is a bug with the internal rendering. I am finishing working on a fix. I'll let you know once the fix is live!

**Comment by David-Moreira** [comment, marked as answer]
Hello @ThankVisualJkseo 
Thank you for reporting this bug. This is now fixed in v1.6.2
